controllers/suite_test.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package controllers
+package controllers_test
```

Comment:
What's the best practice here - put the tests in the same package or *_test? I don't mind it, just want to be consistent

Reply:
`_test` so we are not tempted to test non-public APIs and write sensible public ones.